packages/office-ui-fabric-react/src/components/Dialog/DialogContent.styles.ts

```diff
@@ -100,20 +101,15 @@ export const getStyles = (props: IDialogContentStyleProps): IDialogContentStyles
         color: palette.neutralPrimary,
         margin: '0',
         padding: '16px 46px 24px 24px',
-        fontSize: 20, // TODO: after updating the type ramp this needs reevaluated
-        fontWeight: FontWeights.semibold,
         lineHeight: 'normal'
       },
-      isLargeHeader && [
-        fonts.xxLarge,
-        {
-          color: palette.white,
-          marginBottom: '8px',
-          padding: '22px 24px',
-          fontWeight: FontWeights.semibold
-        }
-      ],
-      isMultiline && fonts.xxLarge
+      isLargeHeader && {
+        fontSize: fonts.xxLarge.fontSize,
+        color: palette.white,
+        marginBottom: '8px',
+        padding: '22px 24px'
+      },
+      isMultiline && { fontSize: fonts.xxLarge.fontSize }
```

Comment:
Is there a reason we only apply the `xxLarge` font size here and not the entire `xxLarge` font variant?

Reply:
Because the `title` section to which this applies is already setting it up to `fonts.xLarge` and does not need a full reapplication of the all fonts.variant styles.